monkey/monkey_island/cc/ui/src/components/ui-components/IslandResetModal.tsx

```diff
@@ -91,15 +89,32 @@ const IslandResetModal = (props: Props) => {
     }
   }
 
-  function resetIsland(url: string, callback: () => void) {
-    auth.authFetch(url)
-      .then(res => res.json())
+  function clearSimulationData(callback: () => void) {
+    auth.authFetch('/api/clear-simulation-data', {method: 'POST'})
       .then(res => {
-        if (res['status'] === 'OK') {
+        if (res.status === 200) {
           callback()
         }
       })
   }
+  function resetAll(callback: () => void) {
+    auth.authFetch('/api/reset-agent-configuration', {method: 'POST'})
+      .then(res => {
+        if (res.status === 200) {
+            auth.authFetch('/api/clear-simulation-data', {method: 'POST'})
+              .then(res => {
+                if (res.status === 200) {
+                    auth.authFetch('/api/island-mode', {method: 'POST', body: '{"mode": "unset"}'})
+                      .then(res => {
+                        if (res.status === 200) {
+                            callback()
```

Comment:
A some code in this file is missing `;`. We should add the eslint inspection for this?